Docs/SupportedAPIs.md

```diff
@@ -57,8 +57,8 @@ Contributions to expand support to unimplemented functionality are always welcom
 | POST   | `/session/:sessionId/touch/up`                      | Supported    | `Session.touchUp()` |
 | GET    | `/session/:sessionId/url`                           | Not supported| `Session.url`       |
 | POST   | `/session/:sessionId/url`                           | Not supported| `Session.url()`     |
-| DELETE | `/session/:sessionId/window`                        | Supported    | Not implemented     |
-| POST   | `/session/:sessionId/window`                        | Supported    | Not implemented     |
+| DELETE | `/session/:sessionId/window`                        | Supported    | `Session.window()`  |
+| POST   | `/session/:sessionId/window`                        | Supported    | `Session.window()`  |
```

Comment:
The swift-webdriver API column isn't quite right for these. 

I don't think there is a method for the `DELETE` version (though we should add one), and the `POST` version should be `Session.focus()`.

Reply:
We should also update the rows for `/session/:sessionId/window/:windowHandle/size` below.